test_config.yaml

```diff
@@ -14,16 +14,16 @@ fileExistenceTests:
   shouldExist: true
   uid: 1002
   gid: 0
-- name: hive-site.xml
+- name: metastore-site.xml
```

Comment:
confused why we want to rename `hive-site.xml` to `metastore-site.xml`

Reply:
can you add more context

Reply:
I'm just renaming the test.

Reply:
We changed the name of the file in #37.

Reply:
So is `hive-site.xml` renamed to `metastore-site.xml`? Would this change need to  be reflected in our environment too?